### PR TITLE
chore: at_onboarding_cli uptake of at_commons 4.0.0

### PR DESCRIPTION
**Pull request by srieteja**
**- What I did**
- build: updated dependencies of at_onboarding_cli
- tests: updated functional tests

**- How I did it**
- build: bumped up versions to the latest version
- tests: 
-- updated test package dependencies
--  replaced deprecated methods with new methods

**- How to verify it**
- passing of static analysis, unit tests and functional tests

**- Description for the changelog**

chore: at_onboarding_cli uptake of at_commons 4.0.0

### CODE REVIEW

File: packages/at_onboarding_cli/pubspec.yaml [hunk 1]
```diff
@@ -21,15 +21,28 @@ dependencies:
...
+   at_chops: ^1.0.7
+   at_client: ^3.0.73
+   at_commons: ^4.0.0
+   at_lookup: ^3.0.44
    at_server_status: ^1.0.3
-   at_utils: ^3.0.15
+   at_utils: ^3.0.16
+
+dependency_overrides:
+  at_server_status:
```
Comment (murali-shris) [inline packages/at_onboarding_cli/pubspec.yaml]: replace with published version

File: packages/at_onboarding_cli/pubspec.yaml [hunk 2]
```diff
@@ -21,15 +21,28 @@ dependencies:
...
+    git:
+      url: https://github.com/atsign-foundation/at_libraries.git
+      ref: trunk
+      path: packages/at_server_status
+
+  at_auth:
```
Comment (murali-shris) [inline packages/at_onboarding_cli/pubspec.yaml]: check with @purnimavenkatasubbu for PR in at_auth. publish and merge

**Review by murali-shris — CHANGES_REQUESTED** [review]

File: packages/at_onboarding_cli/pubspec.yaml (hunk 1 above)
Comment (purnimavenkatasubbu) [reply packages/at_onboarding_cli/pubspec.yaml]: Updated

File: packages/at_onboarding_cli/pubspec.yaml (hunk 2 above)
Comment (purnimavenkatasubbu) [reply packages/at_onboarding_cli/pubspec.yaml]: updated

**Review by murali-shris — APPROVED** [review]

**Pull request merged**